app/views/spotlight/catalog/_delete_resources.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="modal fade" id="delete-resources-modal" tabindex="-1" role="dialog" aria-labelledby="save-modal-label" aria-hidden="true">
+  <div class="modal-dialog">
+  <%= bootstrap_form_for([:delete_resources, current_exhibit], url: delete_resources_exhibit_dlme_bulk_actions_path(current_exhibit), method: 'post') do |f| %>
+    <div class="modal-content">
+      <div class="modal-header">
+        <h4 class="modal-title" id="save-modal-label"><%= t(:'spotlight.bulk_actions.delete_resources.heading') %></h4>
+        <button type="button" class="blacklight-modal-close close" data-dismiss="modal" aria-label="<%= t('blacklight.modal.close') %>">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        <%= render Blacklight::HiddenSearchStateComponent.new(params: search_state.params_for_search) %>
+        <p>
+          <%= t(:'spotlight.bulk_actions.delete_resources.description', count: @response.total) %>
+        </p>
+
+        <div class="alert alert-danger container" role="alert">
+          <div class="row">
+            <div class="col-1">
+              <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-exclamation-triangle" viewBox="0 0 16 16">
+                <path d="M7.938 2.016A.13.13 0 0 1 8.002 2a.13.13 0 0 1 .063.016.146.146 0 0 1 .054.057l6.857 11.667c.036.06.035.124.002.183a.163.163 0 0 1-.054.06.116.116 0 0 1-.066.017H1.146a.115.115 0 0 1-.066-.017.163.163 0 0 1-.054-.06.176.176 0 0 1 .002-.183L7.884 2.073a.147.147 0 0 1 .054-.057zm1.044-.45a1.13 1.13 0 0 0-1.96 0L.165 13.233c-.457.778.091 1.767.98 1.767h13.713c.889 0 1.438-.99.98-1.767L8.982 1.566z"/>
+                <path d="M7.002 12a1 1 0 1 1 2 0 1 1 0 0 1-2 0zM7.1 5.995a.905.905 0 1 1 1.8 0l-.35 3.507a.552.552 0 0 1-1.1 0L7.1 5.995z"/>
+              </svg>
```

Comment:
this svg is the warning triangle in the delete confirmation dialog.  i'd initially tried to render it using just bootstrap classes, but that didn't work for me, so i used their suggested svg.  also happy to revisit this if there's a better way (or if i was just doing something wrong the other way i tried).